### PR TITLE
1250 date parsing exception

### PR DESCRIPTION
**Pull request by zebdelk**

### CODE REVIEW

File: schemas-analyze/src/test/groovy/org/cedar/schemas/analyze/DateInfoSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import static spock.util.matcher.HamcrestMatchers.closeTo
+
+@Unroll
+class DateInfoSpec extends Specification {
+
+  final String analysisAvro = ClassLoader.systemClassLoader.getResourceAsStream('avro/psi/analysis.avsc').text
+
+  def 'extracts expanded date info from date string: #input'() {
+    when:
+    def result = new DateInfo(input, true) // shouldn't matter if start is true or false for these
```
Comment (ajakz) [inline schemas-analyze/src/test/groovy/org/cedar/schemas/analyze/DateInfoSpec.groovy:22]: According to our spec, THIS IS NOT TRUE. These fields need to be extrapolated and they are not.

Comment (zebdelk) [reply schemas-analyze/src/test/groovy/org/cedar/schemas/analyze/DateInfoSpec.groovy:22]: Might need more tests, or comments to make this clearer, but the DateInfo object doesn't care about start or end for the fields being tested in this particular test. The Temporal util class pulls info from different fields for some of the end and instant dates, to match up to the spec.

Comment (ajakz) [reply schemas-analyze/src/test/groovy/org/cedar/schemas/analyze/DateInfoSpec.groovy:22]: I see it now in Temporal:
```
      builder.setEndDayOfYear(endInfo.endDayOfYear == null ? endInfo.dayOfYear : endInfo.endDayOfYear);
      builder.setEndDayOfMonth(endInfo.endDayOfMonth == null ? endInfo.dayOfMonth : endInfo.endDayOfMonth);
      builder.setEndMonth(endInfo.endMonth == null ? endInfo.month : endInfo.endMonth);
```

So extrapolated info occurs later in the flow, got it. :+1:

**Comment by ajakz** [comment]
LGTM except for what is outlined in #35 (and can be addressed later).

**Pull request merged**